### PR TITLE
PTX-2528 add better logging for pods in torpedo

### PR DESCRIPTION
**Pull request by thiguetta**
Signed-off-by: Thiago Gonzaga <thi_gonzaga@yahoo.com.br>

### CODE REVIEW

File: tests/common.go [hunk 1]
```diff
@@ -617,49 +619,51 @@ func getStorageNodes() ([]node.Node, error) {
...
+					TimeBeforeRetry: defaultCmdRetryInterval,
+					Sudo:            true,
+				})
+				if err == nil {
+					runCmd(fmt.Sprintf("chmod 755 %s/oci.log", Inst().BundleLocation), n)
+				}
+			})
+		}
 	})
 }
```
Comment (harsh-px) [inline tests/common.go:661]: looks like this PR has unrelated changes other than just improving pod logging. Let's separate them out.

Comment (thiguetta) [reply tests/common.go]: done
